src/main/java/com/powsybl/network/conversion/server/NetworkConversionException.java

```diff
@@ -1,11 +1,36 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
 package com.powsybl.network.conversion.server;
 
+import java.util.Objects;
+
 /**
  * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
  */
-class NetworkConversionException extends RuntimeException {
+public final class NetworkConversionException extends RuntimeException {
+
+    public enum Type {
+        UNSUPPORTED_FORMAT
```

Comment:
why an enum that has only one value ? if there's not other NetworkConversionException types we don't need to create an enum !

Reply:
not yet. but there will be and it'll be easier to add them.